caselaw/urls.py

```diff
@@ -13,10 +13,12 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from django.conf.urls import url
 from django.contrib import admin
 from django.urls import include, path
 
 urlpatterns = [
     path("", include("legal_db.urls")),
     path("admin/", admin.site.urls),
+    url(r"^markdownx/", include("markdownx.urls")),
```

Comment:
What are these URLs used for?

Reply:
They are required by [markdownx](https://neutronx.github.io/django-markdownx/getting_started/) and other packages has a similar requirement for showing a preview of the resulting format while editing the markdown field.

Reply:
sounds good